README.md

```diff
@@ -59,6 +59,7 @@ packages via the command line.
   * [Fedora package] stable release: `# dnf install freeorion`
   * [Void package] stable release: `# xbps-install freeorion`
   * [ArchLinux package] stable release
+  * [openSUSE] stable release: '# zypper in freeorion'
```

Comment:
Link isn't displayed, also command isn't formatted: 

https://github.com/freeorion/freeorion/blob/558e21424fb80fbc0df8115017fce60abe2ab9ba/README.md

Reply:
The command isn't formated because of using `'` instead of `` ` ``
No idea about the [openSUSE] not displaying link (maybe it should be [OpenSUSE] or the keyword definition is missing, wherever it should be).

Reply:
@Oberlus Links are listed in the bottom of document.

Reply:
Ah, then it's easy: [openSUSE] is not defined in the document.